CHANGELOG.md

```diff
@@ -7,7 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-* [PR-14](https://github.com/ITK-Leantime/leantime-dataexport/pull/15)
+* [PR-17](https://github.com/ITK-Leantime/leantime-dataexport/pull/17)
+  Update import paths to comply with leantime 3.2.0
+  Update from php8.1 -> php8.3
+  Update leantime dependency
+  Commit composer.lock
```

Comment:
Why is `composer.lock` committed? Do we do that in other plugins?

Reply:
Hmm, there might be another way to do it, but the actions didn't seem to update leantime and kept installing an old version of leantime and found errors doing code analysis.

Reply:
It's customary to *not* include `composer.lock` in something that is not a standalone project. However, locking the versions of development dependencies may make sense to prevent that updated tools suddenly start complaining about something that wasn't an issue before.
But I have a feeling that if we *don't* always use the latest versions of the tools, i.e. lock the versions, we'll (almost) never update them and therefore not benefit from any improvements made to them.

If you change `composer install` to `composer update` in <https://github.com/ITK-Leantime/leantime-dataexport/blob/main/README.md> you should always get the latest version of the tools.

Reply:
Hmm … The Actions log (e.g. <https://github.com/ITK-Leantime/leantime-dataexport/actions/runs/9793645307/job/27050497624#step:3:149>) says 

```
No composer.lock file present. Updating dependencies to latest instead of installing from lock file. See https://getcomposer.org/install for more information.
```

and reports the versions that are actually installed. Are some versions not as expected?